datalad/distribution/create_sibling.py

```diff
@@ -92,7 +92,7 @@ def _create_dataset_sibling(
         ds_target_pushurl = target_pushurl.replace('%RELNAME', ds_name) \
             if target_pushurl else ds_sshurl
 
-    lgr.info("Creating target dataset {0} at {1}".format(
+    lgr.info("Considering to create a target dataset {0} at {1}".format(
```

Comment:
It is not attempting anything yet -- just doing some kind of analysis while considering to create.   "Preparing" is a viable possibility... but not better enough worth rerunning the tests ;)  Since otherwise is ok, I will merge and then will let others to improve wording